code/modules/mob/living/living_say.dm

```diff
@@ -475,6 +472,12 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 		var/replacement = tts_message[length_regex.index]+tts_message[length_regex.index]+tts_message[length_regex.index]
 		tts_message = replacetext(tts_message, length_regex.match, replacement, length_regex.index)
 
+	///removes repeated consonants at the start of a word: ex: sss
```

Comment:
```suggestion
	// removes repeated consonants at the start of a word: ex: sss
```

Doc comments don't show here

Reply:
Looks like I saw this late, but jsyk this comment was copied from the above one, that should be edited too for consistency